localstack-core/localstack/config.py

```diff
@@ -433,6 +433,9 @@ def in_docker():
 # When enabled it triggers specialised workflows for the debugging.
 LAMBDA_DEBUG_MODE = is_env_true("LAMBDA_DEBUG_MODE")
 
+# path to the lambda debug mode configuration file.
+LAMBDA_DEBUG_MODE_CONFIG_PATH = os.environ.get("LAMBDA_DEBUG_MODE_CONFIG_PATH")
```

Comment:
Can we add all new environment variables to `CONFIG_ENV_VARS` in `config.py` at the bottom such that they work using the CLI?